integration-tests/server/test/index.test.js

```diff
@@ -71,9 +71,10 @@ describe('@overture-stack/arranger-server', () => {
       });
     }
 
-    console.log('===== Starting arranger app for test =====');
     try {
       const router = await Arranger({
+        // This may be useful when troubleshooting tests
+        // enableLogs: true,
```

Comment:
added this functionality to the server, for dev purposes (used to be EnvVar only which requires more insight on the internals of the server)